### DISCUSSION TITLE
Blockquotes formatting  not working after lists

### DISCUSSION THREAD

**Discussion by robozb** [mkdocs/mkdocs#3267] [Q&A]
![image](https://github.com/mkdocs/mkdocs/assets/8656998/25dd4bb2-78f1-4d58-9dae-fa0ef1f454bf)

**Comment by pawamoy** [comment]
As a workaround you can use fenced code blocks instead of indentation:

````md
```
block1
```

- item
- item
- item

```
block2
```
````

**Comment by facelessuser** [comment]
This is true for all Markdown parsers. If under a list you have to indent under the list. Let's see what GitHub does.

```
- item

    block2
```

- item

    block2

But if we indent under the list:

```
- item

        block2
```

- item

        block2

**Comment by robozb** [comment]
OK, I understand. Thank you so much!
